gcc/opts.cc

```diff
@@ -2771,7 +2771,11 @@ common_handle_option (struct gcc_options *opts,
 	= parse_sanitizer_options (arg, loc, code,
 				   opts->x_flag_sanitize_recover, value, true);
       break;
-
+    case OPT_afmv_:
```

Comment:
OPT_afmv_ needs to be declared somewhere,
The code will not compile as it is.

Reply:
Which file are you talking about, OPT_fsanitize_recover_ and many other variables used in case are not declared i n header file, nor are they declared in the current opts.cc file, where do you want me to declare them.

Reply:
In https://github.com/gcc-mirror/gcc/blob/master/gcc/langhooks.h
It mentions that there's supposed to be an options.h file that contains the different OPT_ enum flags.
I haven't been able to find the source where it's located but from what I've searched that enum flag isn't declared anywhere.